src/Entity/Storage/DeveloperStorage.php

```diff
@@ -119,15 +119,9 @@ protected function entityController(): EdgeEntityControllerInterface {
   public function loadMultiple(array $ids = NULL) {
     $entities = parent::loadMultiple($ids);
     if ($ids) {
-      $entitiesByDeveloperId = [];
-      foreach ($entities as $entity) {
-        // It could be an integer if developer UUID has been used as as an id
-        // instead of the email address.
-        if (is_object($entity)) {
-          /** @var \Drupal\apigee_edge\Entity\DeveloperInterface $entity */
-          $entitiesByDeveloperId[$entity->getDeveloperId()] = $entity;
-        }
-      }
+      // It could be an integer if developer UUID has been used as as an id
+      // instead of the email address.
+      $entitiesByDeveloperId = array_filter($entities, 'is_object');
```

Comment:
Replaced `foreach() `with `array_filter()` that does not spam the method's scope with remnant variables.